pages/blog/escrow.js

```diff
@@ -0,0 +1,197 @@
+              contracts do not require any third-party control and run solely on
+              the basis of pre-programmed logic which is overseen by a
+              distributed, decentralized network of computers that runs on
+              <a
+                className="blog_link"
+                href="https://www.investopedia.com/terms/b/blockchain.asp#what-is-blockchain"
+              >
+                blockchain
+              </a>
+              . It is a computer protocol that is used to digitally and
+              automatically facilitate, verify, and enforce the terms of a
+              contract. The main benefit of the Escrow smart contract is to
+              reduce the dependency of intermediates, centralized arbitrators to
+              help reduce fraud transactions. The program automatically
+              validates a condition and determines whether an asset will go to
+              one person or back to the person who sent it. It is interesting to
+              know that during the process, the
+              <a
+                className="blog_link"
+                href="https://searchcio.techtarget.com/definition/distributed-ledger"
+              >
+                distributed ledger
+              </a>{' '}
+              can also store and make a copy of the document which provides
+              added security and immutability.
+            </p>
+          </section>
+          <section className="blog__section">
+            <h2>Main Advantages of using Escrow Smart Contracts</h2>
+            <div>
+              <p>
+                <strong>
+                  Main Advantages of using Escrow Smart Contracts Smart Contract
+                  enforcement:
+                </strong>{' '}
+                Smart contracts are enforced the same way as normal programs are
+                done. Smart contracts are similar to real-world contracts but
+                not exactly. In the real world, if a contract is broken you can
+                take the person to court but in the case of smart contracts,
+                there are predefined rules that are defined by parties and
+                enforced by the blockchain. It’s smart contract enforcement.
+              </p>
+              <p>
+                <strong>
+                  Arbitration Decentralized (reduce conflict interest) with{' '}
+                  <a className="blog_link" href="https://kleros.io/">
+                    Kleros
+                  </a>
+                  /
+                  <a
+                    className="blog_link"
+                    href="https://aragon.org/blog/aragon-network-jurisdiction-part-1-decentralized-court-c8ab2a675e82"
+                  >
+                    Aragon
+                  </a>
+                </strong>
+                : The arbitration is decentralized and if any conflict of
+                interest arises, it is being dealt with{' '}
```

Comment:
no space after the Kleros link @amitg01 

![Screen Shot 2020-10-27 at 9 47 14 AM](https://user-images.githubusercontent.com/72858592/97277966-6d3f4e80-1839-11eb-8f3d-11f59f6ed310.png)